### PR TITLE
heketi:heketi topology sample add

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

### CODE REVIEW

**Comment by MohamedAshiqrh** [comment]
https://github.com/heketi/heketi/issues/370

File: client/cli/go/topology-sample.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    "node": {
+                        "hostnames": {
+                            "manage": [
+                                "192.168.10.100"
+                            ],
+                            "storage": [
+                                "192.168.10.100"
+                            ]
+                        },
+                        "zone": 0
```
Comment (lpabon) [inline client/cli/go/topology-sample.json]: Zone 0 is not an accepted value.  1 or above

File: client/cli/go/topology-sample.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                    "node": {
+                        "hostnames": {
+                            "manage": [
+                                "192.168.10.102"
+                            ],
+                            "storage": [
+                                "192.168.10.102"
+                            ]
+                        },
+                        "zone": 0
```
Comment (lpabon) [inline client/cli/go/topology-sample.json]: Here too

File: client/cli/go/cmds/volume.go [hunk 3]
```diff
@@ -142,7 +142,7 @@ var volumeCreateCommand = &cobra.Command{
 		if kubePv && kubePvEndpoint == "" {
 			fmt.Fprintf(stderr, "--persistent-volume-endpoint must be provided "+
 				"when using --persistent-volume\n")
-			return fmt.Errorf("Missing endpoing")
+			return fmt.Errorf("Missing endpoint")
```
Comment (lpabon) [inline client/cli/go/cmds/volume.go]: This change is also in #373 , you probably want to cancel that one or remove this one.

**Comment by MohamedAshiqrh** [comment]
Done.

**Pull request merged**